fe/src/main/java/org/apache/doris/transaction/TransactionState.java

```diff
@@ -321,4 +330,11 @@ public LoadJobSourceType getSourceType() {
     public Map<Long, PublishVersionTask> getPublishVersionTasks() {
         return publishVersionTasks;
     }
+
+    public boolean isPublishTimeout() {
+        // timeout is between 3 to 10 seconds.
+        long timeoutMillis = Math.min(Config.publish_version_timeout_second * publishVersionTasks.size() * 1000, 10000);
```

Comment:
set 10000 to a config, and the minimum value should be 5min

Reply:
Set it as Config.load_straggler_wait_second